platform/security/policy/security-policy-context/src/main/java/org/codice/ddf/security/policy/context/impl/Policy.java

```diff
@@ -90,4 +91,19 @@ public CollectionPermission getAllowedAttributePermissions() {
         }
         return new ArrayList<ContextAttributeMapping>(attributeMappings);
     }
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("Context Path: " + contextPath +
```

Comment:
This just hurts. You've a `StringBuilder`; don't do all this String concatenation.